src/Microsoft.Dnx.Host.Mono/EntryPoint.cs

```diff
@@ -74,10 +75,24 @@ public static int Main(string[] arguments)
             Environment.SetEnvironmentVariable(EnvironmentNames.AppBase, arguments[appbaseIndex + 1]);
         }
 
+        string operatingSystem, osVersion, architecture;
+        GetOsDetails(out operatingSystem, out osVersion, out architecture);
+
+        var bootstrapperContext = new BootstrapperContext
+        {
+            OperatingSystem = operatingSystem,
+            OsVersion = osVersion,
+            Architecture = architecture,
+            RuntimeDirectory = Path.GetDirectoryName(typeof(EntryPoint).Assembly.Location),
```

Comment:
Then it was always broken - because we already use this: https://github.com/aspnet/dnx/pull/2769/files#diff-f79c267a7f222c20dba5339e1498a6d3R59. Where can we get the information from in the case you describe?